server/metadata/users.go

```diff
@@ -61,10 +62,10 @@ func (u *UserSubspace) GetUserMetadata(ctx context.Context, tx transaction.Tx, n
 		return nil, err
 	}
 	var row kv.KeyValue
-	for it.Next(&row) {
+	if it.Next(&row) {
```

Comment:
The below `UpdateUserMetadata` still needs fixing.

Reply:
I did not follow your suggestion here. please elaborate.